request/unpacker/base.go

```diff
@@ -177,7 +178,7 @@ func (b *BaseUnpacker) parseResponseElements() error {
 		return nil
 	}
 
-	if b.httpResponse.Header.Get("Content-Type") != "application/json" {
+	if !strings.Contains(b.httpResponse.Header.Get("Content-Type"), "application/json") {
```

Comment:
> `HasPrefix` maybe better?

I think the semicolon is just a separator, and the data are not in a certain sequence (RFC 1867 & 2388). So I used Contains instead of HasPrefix.

Reply:
OK